Samples~/my-unity-crasher/BugSplatOptions.asset

```diff
@@ -12,9 +12,9 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 485e0e2b32558c54fb70b32dea6b7c38, type: 3}
   m_Name: BugSplatOptions
   m_EditorClassIdentifier: 
-  Database: fred
-  Application: 
-  Version: 
+  Database: unity_plugin_test
+  Application: BugSplatDemo
+  Version: 1.0 (3)
```

Comment:
```suggestion
  Database: fred
  Application: 
  Version: 
```

Reply:
This is related to the [comment](https://github.com/BugSplat-Git/bugsplat-unity/pull/74/files#r1035091838) above.